go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/giantswarm/kvm-operator
 go 1.14
 
 require (
-	github.com/giantswarm/apiextensions v0.3.9
+	github.com/giantswarm/apiextensions v0.3.10-0.20200513205053-0bf9395d65eb
```

Comment:
This will be changed to 0.3.10 when https://github.com/giantswarm/apiextensions/pull/473 is released.